automation/addLabelsOnPullRequest.automation.js

```diff
@@ -65,31 +65,30 @@ exports.addMergedLabel = async context => {
 		const issueLabels = await listIssueLabels(context);
 
 		const foundApproveLabel = issueLabels.data.filter(
-			label => label.name === ':white_check_mark: Approved'
+			label => label.name === ":white_check_mark: Approved"
 		);
 		const foundWIPLabel = issueLabels.data.filter(
-			label => label.name === ':construction: WIP'
+			label => label.name === ":construction: WIP"
 		);
 
-		addLabel([':sparkles: Merged:'], context);
+		addLabel([":sparkles: Merged:"], context);
 
 		if (foundApproveLabel.length > 0) {
-			removeLabel([':white_check_mark: Approved'], context);
+			removeLabel([":white_check_mark: Approved"], context);
 		}
 
 		if (foundWIPLabel.length > 0) {
-			removeLabel([':construction: WIP'], context);
+			removeLabel([":construction: WIP"], context);
 		}
-		console.log(title);
 
 		if (
-			title.includes('WIP') ||
-			title.includes('Work In Progress') ||
-			title.includes('work in progress') ||
-			title.includes(':construction:')
+			title.includes("WIP") ||
+			title.includes("Work In Progress") ||
+			title.includes("work in progress") ||
+			title.includes(":construction:")
 		) {
 			const params = context.pullRequest({
-				title: `${title.replace(':construction:', '')}`,
+				title: `${title.replace(":construction:", "")}`,
```

Comment:
Codacy has a fix for the issue: [Replace `↹↹↹↹` with `········`](https://app.codacy.com/gh/aayushchugh/bot/pullRequest?prid=10060881)
```suggestion
        title: `${title.replace(":construction:", "")}`,
```